src/WebJobs.Script.WebHost/Management/InstanceManager.cs

```diff
@@ -122,6 +132,12 @@ public bool StartAssignment(HostAssignmentContext context)
             }
         }
 
+        private void SpecializeForEmptyScmPackage()
+        {
+            _logger.LogInformation("Specialized with currently empty scm package");
+            _webHostEnvironment.FlagAsSpecializedAndReady();
```

Comment:
these flags are fragile. try not to add more ways of setting them. Use the call already being made in Assign()